README.md

```diff
@@ -90,6 +90,38 @@ ros2 launch l3xz control.py
 **Note**: MikroTik [RBMetal5SHPn](https://mikrotik.com/product/RBMetal5SHPn).
 
 #### Cyphal Configuration
+##### How-to-config via `yakut`
+Configure `can0`:
+```bash
+sudo ./setup_slcan.sh --remove-all --basename can --speed-code 5 /dev/serial/by-id/usb-Zubax_Robotics_Zubax_Babel_*-if00
+```
+Configure `yakut`:
+```bash
+python3 -m pip install yakut
+yakut compile https://github.com/OpenCyphal/public_regulated_data_types/archive/refs/heads/master.zip
+. setup_yakut.sh
+```
+Dump all write-able registers into l3xz-cyphal-config.yaml
+```bash
+y rl 11,12,13,14,15,16,20,30,40,50 | y --yaml rb --only=mp > config/cyphal-config.yaml
```

Comment:
This is not wrong but you could also say it like this:

```suggestion
y rl 11-17,20,30,40,50 | y --yaml rb --only=mp > config/cyphal-config.yaml
```

Reply:
I saw this but I'm not happy about the syntax. Personally I'd expect to use 11-16 to catch nodes 11 to 16.

Reply:
Sometimes I feel you love formalism too much. Well, for me this user-interface is counter-intuitive but I accept your way of designing it. At any rate, I would not advocate changing it purely from a backwards-compatibility perspective. In turn I hope you'll forgive me if I choose to manually enumerate all IDs I want to capture :grin: